packages/cashier/src/components/cashier-notifications/cashier-notifications-icon.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import CashierNotifications from './cashier-notifications';
+
+// This component is used in general-store
```

Comment:
@hamid-deriv It seems this component is not doing anything 🤔 Couldn’t we just use `CashierNotifications` instead? 🤔

Reply:
Nive tip. Removed.